cmd/main.go

```diff
@@ -265,6 +270,12 @@ func main() {
 			"Failed to parse RELEASE_IMAGES json %s", Options.ReleaseImages)
 	}
 
+	var releaseSourcesArray = models.ReleaseSources{}
+	if Options.ReleaseSources != "" {
```

Comment:
If RELEASE_IMAGES and RELEASE_SOURCES are both given, is that valid?

Reply:
`RELEASE_IMAGES` is our current configuration for release images that could be installed, and `RELEASE_SOURCES` is a new configuration that specifies which release images the go routine should collect, basically they are undependable on one  another, but the fact if one of them missing (or both) there won't be nothing to merge later on